docs/man

```diff
@@ -0,0 +1,41 @@
+# The man Command
+
+## Summary 
+The `man` command in Linux is used to display the user manual of any command that we can run on
+the terminal. It provides a detailed view of the command which includes NAME, SYNOPSIS,
+DESCRIPTION, OPTIONS, EXIT STATUS, RETURN VALUES, ERRORS, FILES, VERSIONS, EXAMPLES,
+AUTHORS and SEE ALSO.
+https://www.nielit.gov.in/gorakhpur/sites/default/files/Gorakhpur/Alevel_unix_27april20_AKM.pdf
+
+Every manual is divided into the following sections:
+
+Executable programs or shell commands
```

Comment:
This list of sections could be presented as bullets by placing an asterisk before each item.